libs/expo/betterangels/src/lib/screens/Client/Docs/Documents.tsx

```diff
@@ -0,0 +1,75 @@
+import { FolderIcon, FolderOpenIcon } from '@monorepo/expo/shared/icons';
+import { Colors, Radiuses, Spacings } from '@monorepo/expo/shared/static';
+import { Accordion, FileCard } from '@monorepo/expo/shared/ui-components';
+import { useState } from 'react';
+import { View } from 'react-native';
+import { ClientDocumentType, Maybe } from '../../../apollo';
+import { DocumentModal } from '../../../ui-components';
+
+interface IDocumentsProps {
+  expanded: undefined | string | null;
+  setExpanded: (expanded: undefined | string | null) => void;
+  data: ClientDocumentType[];
+  clientId: string;
+  title: 'Other' | 'Doc-Ready' | 'Consent Form';
```

Comment:
I think the three sections are: doc-ready, forms, and other. 
```suggestion
  title: 'Other' | 'Doc-Ready' | 'Forms';
```